Seen in ppc64el builds for multiple releases:

  In file included from /<<PKGBUILDDIR>>/obj-powerpc64le-linux-gnu/gfx/thebes/Unified_cpp_gfx_thebes0.cpp:65:
  In file included from /<<PKGBUILDDIR>>/gfx/thebes/gfxAlphaRecovery.cpp:12:
  /<<PKGBUILDDIR>>/third_party/xsimd/include/xsimd/types/xsimd_vsx_register.hpp:67:61: error: cannot use 'long' with '__vector bool'
     67 |         XSIMD_DECLARE_SIMD_BOOL_VSX_REGISTER(unsigned long, long);
        |                                                             ^
  /<<PKGBUILDDIR>>/third_party/xsimd/include/xsimd/types/xsimd_vsx_register.hpp:68:52: error: cannot use 'long' with '__vector bool'
     68 |         XSIMD_DECLARE_SIMD_BOOL_VSX_REGISTER(long, long);
        |                                                    ^
  /<<PKGBUILDDIR>>/third_party/xsimd/include/xsimd/types/xsimd_vsx_register.hpp:70:54: error: cannot use 'long' with '__vector bool'
     70 |         XSIMD_DECLARE_SIMD_BOOL_VSX_REGISTER(double, long);
        |                                                      ^

So we can't use "long", but "long long" works just fine. (*shrug*)

The first two declarations are dropped, because if we change the second arg
to "long long", we get

  .../xsimd_vsx_register.hpp:LL:CC: warning: use of 'long' with '__vector' is deprecated [-Wdeprecated]

--- a/third_party/xsimd/include/xsimd/types/xsimd_vsx_register.hpp
+++ b/third_party/xsimd/include/xsimd/types/xsimd_vsx_register.hpp
@@ -64,10 +64,8 @@ namespace xsimd
         XSIMD_DECLARE_SIMD_BOOL_VSX_REGISTER(short, short);
         XSIMD_DECLARE_SIMD_BOOL_VSX_REGISTER(unsigned int, int);
         XSIMD_DECLARE_SIMD_BOOL_VSX_REGISTER(int, int);
-        XSIMD_DECLARE_SIMD_BOOL_VSX_REGISTER(unsigned long, long);
-        XSIMD_DECLARE_SIMD_BOOL_VSX_REGISTER(long, long);
         XSIMD_DECLARE_SIMD_BOOL_VSX_REGISTER(float, int);
-        XSIMD_DECLARE_SIMD_BOOL_VSX_REGISTER(double, long);
+        XSIMD_DECLARE_SIMD_BOOL_VSX_REGISTER(double, long long);
 
 #undef XSIMD_DECLARE_SIMD_BOOL_VSX_REGISTER
     }
